src/cr/cube/cubepart.py

```diff
@@ -291,16 +295,23 @@ def columns_dimension_type(self):
     def columns_margin(self):
         return np.array([column.margin for column in self._matrix.columns]).T
 
+    @lazyproperty
+    def columns_moe(self):
+        """Returns the margin of error (MoE) for col percentages
+        `moe = Z_975 * 100 * std_error` (the * 100 part accounts for percentages)
+        """
```

Comment:
Rename to `column_percentages_moe` so it is explicit and also appears immediately following `column_percentages`, both here in the code and in the documentation.

Prefer `"float margin of error (MoE) for column percentages."` for the summary line. A property, like an attribute, doesn't "return" anything, it "is" some value. Although there are many examples (like the next property below) left to clean up, we reserve the `"Returns ...` prefix for methods and functions.

Blank line before continuation when the docstring extends beyond the first line: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

`moe = Z_975 * 100 * std_error` just restates the implementation. This docstring appears in the documentation and so should communicate to the user what they are getting from (the contract for) this property, _in user-space terms_. It should also probably state that 3.5% is represented as 3.5 (and not .035). Also it should probably mention that the return value can be NaN and under what circumstances. And we should state clearly whether a returned NaN is a `float("NaN")` or `np.nan` because those two have different behaviors in certain comparisons.

The `(the * 100 ...)` bit can appear as a comment and seems useful. Comments speak in _developer-space_ and topics related to the implementation are appropriate there.